src/Headers/ContentSecurityPolicy/Nonce.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types = 1);
+
+namespace Pd\SecurityHeaders\Headers\ContentSecurityPolicy;
+
+final class Nonce implements IValue
```

Comment:
Jenom takový obecný dotaz k `nonce`. Používáš to někde? Pokud ano, tak jakým způsobem ty `nonce` generuješ a jak je potom dostáváš do šablon, abys to potom zapsal k tagům těch skriptů a stylů?

Reply:
Nepoužívám, přemýšlel jsem na nějakým Nonce generátorem. Projekty by si ho museli předat buď do komponent, které používají nonce, případně přímo do template factory.

Reply:
Případně je možno inspirovat se u novějších verzí Nette, kde existuje makro `n:nonce`. Nevím přesně, jak je to vyřešené, ale v HTTP extension je uděláno to, že je možné v NEONu si nastavit, aby se používalo nonce, v takovém případě se přímo v tomto rozšíření nastavuje to generování `nonce` a odesílání této hlavičky. V `nette/application` se potom definuje to makro `n:nonce`. A v `TemplateFactory` se posílá do šablony ta vygenerovaná hodnota `nonce` z hlavičky.

Viz:
https://github.com/nette/http/blob/master/src/Bridges/HttpDI/HttpExtension.php#L104-L109
https://github.com/nette/application/blob/master/src/Bridges/ApplicationLatte/UIMacros.php#L46
https://github.com/nette/application/blob/master/src/Bridges/ApplicationLatte/TemplateFactory.php#L122-L123